code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -412,12 +412,26 @@
 
 /obj/item/weapon/robot_module/combat/New()
 	src.modules += new /obj/item/device/flash/cyborg(src)
+	src.modules += new /obj/item/weapon/restraints/handcuffs/cable/zipties/cyborg(src)
 	src.modules += new /obj/item/borg/sight/thermal(src)
-	src.modules += new /obj/item/weapon/gun/energy/laser/cyborg(src)
-	src.modules += new /obj/item/weapon/gun/energy/plasmacutter(src)
+	src.modules += new /obj/item/weapon/gun/energy/gun/cyborg(src)
+	src.modules += new /obj/item/weapon/pickaxe/drill/jackhammer(src)

```

Comment:
I really don't understand this; it's so strange. I realize they had a plasmacutter, before--I just don't really understand why.